.storybook/2.DOCUMENTATION.stories.mdx

```diff
@@ -14,18 +14,17 @@ Some general documentation best practices to follow:
 
 - Put yourself in the shoes of another developer trying to use the component you just created for the first time
 - Write a brief description of the component and what it's used for in the `README.mdx` file
-- Display the component's api using the `<ArgsTable of={YourComponent} />` component from storybook docs
-- Use the [controls](https://storybook.js.org/addons/@storybook/addon-controls/) api over knobs
-- Use the updated `argType` [actions](https://storybook.js.org/docs/react/essentials/actions) api over importing the actions plugin directly
-- Check the accessibility of the component using the **Accessibility** tab
+- Display the component's api using the `<ArgsTable of={YourComponent} />` component from storybook docs. Add descriptions of each prop by using jsDoc style comments in the `propTypes`.
```

Comment:
Tiny nit but I think we should uppercase `API`.  Please don't hurt me.

Reply:
Agreed! We should have a consistent terminology and standard. Have updated all instance of lowercase "api" to "API"